packages/ui-admin/src/app/routes/NetPromotingScore/NPS.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode } from 'react'
```

Comment:
Why are the components located in the routes dir ? 
They are not full screen pages, they should reside in a dedicated NPS dir, next to routes, not inside routes

Reply:
I think the code was placed there, because the time will be initialized when the routes are loaded. 

I will move this logic where the code is initialized for the first time. 

Do we have a place for this ?

Reply:
as mentioned, you should create a NPS directory next to the routes dir. similar to the StatusBar directory